### DISCUSSION TITLE
ignore this its for video tutorial

### DISCUSSION THREAD

**Discussion by Gosto70** [community/community#136184] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### **Top area** 
its body discussion
sorry this its not for discussion its video tutorial

**Comment by wechuck** [comment, marked as answer]
i answered this

**Comment by Dhrubojyot** [comment]
Response #2

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/136185#discussioncomment-10386816) for reference to why this discussion was closed and locked. Thanks!
